opentelemetry-api/setup.py

```diff
@@ -40,6 +40,15 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
     ],
+    entry_points={
+        "console_scripts": [
+            "opentelemetry-auto-instrument = "
+            "opentelemetry.auto_instrument.auto_instrument:run"
+        ],
+        "opentelemetry_patcher": [
+            "no_op_patcher = opentelemetry.patcher.no_op_patcher:NoOpPatcher"
```

Comment:
if the `NoOpPatcher` gets removed, this could be removed as well.

Reply:
Removed